src/mmw/package.json

```diff
@@ -33,6 +33,7 @@
     "leaflet": "0.7.3",
     "leaflet-draw": "git://github.com/michaelguild13/Leaflet.draw#e2fe3a4",
     "leaflet.locatecontrol": "0.43.0",
+    "leaflet-plugins": "git://github.com/azavea/leaflet-plugins",
```

Comment:
I think he means this change https://github.com/azavea/leaflet-plugins/commit/d59d90c3bbe69805197ac1ac8c79a657dc03b53b. That would be a good contribution to make back to the leaflet-plugins project.